src/app/app-interactive-map/store/interactive-map.reducers.ts

```diff
@@ -263,6 +278,18 @@ export function interactiveMapReducer(
           const value = card[operationTarget];
           // @ts-ignore
           const result = operationFunction(value, item);
+          if (operationTarget === 'addedReactions') {
+            const addedReaction = addedReactionToReaction(<AddedReaction>item);
+            if (!card.model.reactions.includes(addedReaction)) {
+              card.model.reactions.push(addedReaction);
+            }
+            const metabolites_to_add = (<AddedReaction>item).metabolites_to_add;
+            for (let i = 0; i < metabolites_to_add.length; i++) {
+              if (card.model.metabolites.indexOf(metabolites_to_add[i]) === -1) {
```

Comment:
Could use `includes` here like above

Reply:
I can check it again, but I was having problems, because includes was always true, that's why I changed to indexOf.